robot/basestation/stream_capture.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+from subprocess import Popen, PIPE
+import os
+import datetime
+import flask
+from flask import jsonify, make_response
+import threading
+
+import ros_utils
+from ros_utils import fetch_ros_master_ip
+
+def start_recording_feed(stream):
+    recording_log_msg = 'recording feed of ' + stream
+    formatted_date = datetime.datetime.now().strftime("%I:%M:%S_%B_%d_%Y")
+    filename = stream + '_' + formatted_date
+    save_video_dir = 'rover_stream/' + stream
+    stream_url = "http://" + fetch_ros_master_ip() + ":8080/stream?topic=/cv_camera/image_raw"
+    return feed_connection_check(stream, filename, save_video_dir, stream_url, recording_log_msg)
+
+def stop_recording_feed(stream):
+    recording_log_msg = 'saved recording of ' + stream
+    proc_video[stream].communicate(b'q')
+    return jsonify(recording_log_msg=recording_log_msg)
+
+def feed_connection_check(stream, filename, save_video_dir, stream_url, recording_log_msg):
```

Comment:
I feel like this function has a bit too many parameters, like `save_video_dir`, could that not be a constant?
As for `stream` and `stream_url` could `stream_url` be defined based of `stream`?